ccl/changefeedccl: TestChangefeedErrors failed

ccl/changefeedccl.TestChangefeedErrors [failed](https://tanzanite.cluster.engflow.com/invocations/default/72d34277-db1b-457f-8b15-0e96f4978826?testReportRun=17&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on release-24.2 @ [d6523e58ab3c09998535e88646ca16c1ead1ae4e](https://github.com/cockroachdb/cockroach/commits/d6523e58ab3c09998535e88646ca16c1ead1ae4e):


```
=== RUN   TestChangefeedErrors
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestChangefeedErrors817735137
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_test.go:5457: failed assert error: operation "expect-err" timed out after 5.005s (given timeout 5s): expected error 'failed to parse certificate data', got: pq: query execution canceled
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestChangefeedErrors817735137
--- FAIL: TestChangefeedErrors (9.66s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=17</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #123079 ccl/changefeedccl: TestChangefeedErrors failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-master branch-release-24.1]
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedErrors.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-40258

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -5470,7 +5470,7 @@ func TestChangefeedErrors(t *testing.T) {
 		`CREATE CHANGEFEED FOR foo INTO $1 WITH format='experimental_avro', confluent_schema_registry=$2`,
 		`kafka://nope`, `https://schemareg-nope/?ca_cert=!`,
 	)
-	sqlDB.ExpectErrWithTimeout(
+	longTimeoutSQLDB.ExpectErrWithTimeout(
 		t, `failed to parse certificate data`,
 		`CREATE CHANGEFEED FOR foo INTO $1 WITH format='experimental_avro', confluent_schema_registry=$2`,
 		`kafka://nope`, `https://schemareg-nope/?ca_cert=Zm9v`,

